app/Services/Manager.php

```diff
@@ -226,30 +253,55 @@ public function getUserActivity($id, $model)
         }
 
         if ($signup && $signup->reportback) {
-            // Provide the admin URL to the reportback.
-            $signup->reportback->admin_url = env('PHOENIX_URI') . '/admin/reportback/' . $signup->reportback->id;
-
-            // Format the update timestamp
-            $signup->reportback->updated_at = new Carbon($signup->reportback->updated_at);
-            $signup->reportback->updated_at = $signup->reportback->updated_at->format('Y-m-d');
-
-            // Set flagged status to 'pending' if it is NULL, otherwise use bool value
-            if (! isset($signup->reportback->flagged)) {
-                $signup->reportback->flagged = 'pending';
-            } elseif ($signup->reportback->flagged) {
-                $signup->reportback->flagged = 'flagged';
-            } else {
-                $signup->reportback->flagged = 'approved';
-            }
-
-            // Return the reportback.
-            return $signup->reportback;
+            return $this->formatReportback($signup);
         }
 
         // If the user has no activity for this competition or waiting room.
         return null;
     }
 
+    public function getUsersActivity($ids, $model)
```

Comment:
Might be useful to add the `$size` argument in this function like it is in https://github.com/DoSomething/gladiator/blob/00b1b5a34af89a78ff3d3bc9c6f05b31bfc7e0f8/app/Repositories/DatabaseUserRepository.php#L143